### PR TITLE
Task/edcp 121

### PR DESCRIPTION
**Pull request by gali-paz**
finished fixing redirects issues - solved target, ssl and port comparison

### CODE REVIEW

File: wafw00f/lib/evillib.py [hunk 1]
```diff
@@ -331,27 +335,25 @@ def request(self, method='GET', path=None, usecache=True,
...
                                 target = self.target
-                            if port is None:
-                                port = self.port
                             if not path.startswith('/'):
                                 path = '/' + path
-                            if (target, port, ssl) == (self.target, self.port, ssl):
-                                r = self.request(method, path, usecache, cacheresponse,
-                                                 headers, comingfromredir=True)
-                            else:
+                            if target.replace('www.', '') != self.target.replace('www.', ''):
```
Comment (doronlh) [inline wafw00f/lib/evillib.py]: Shouldn't the regex be more specific for this and only replace www if it exists at the beginning of the target string? Something like:
```
import re
re.sub('^www.', '', target) != re.sub('^www.', '', self.target):
```

Comment (gali-paz) [inline wafw00f/lib/evillib.py]: you're right, I've changed it

Comment (doronlh) [inline wafw00f/lib/evillib.py]: I think you forgot to add in the `import re`

Comment (gali-paz) [inline wafw00f/lib/evillib.py]: It was already there

Comment (doronlh) [inline wafw00f/lib/evillib.py]: ah, my bad. The diff didn't show it. I should have looked at the whole file.

**Pull request merged**